gvm/transform/object/responses.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2020 Greenbone Networks GmbH
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from dataclasses import dataclass
+from lxml import etree
+from .classes import (
+    Role,
+    Task,
+    Target,
+    PortList,
+    Config,
+    Scanner,
+    Preference,
+    User,
+    Report,
+    Group,
+)
+
+
+@dataclass
+class Response:
+    """
+    standard Python Response Object
+    """
+
+    response_name: str
+    status: int
+    status_text: str
+
+    def __init__(self, root: etree.Element):
+        self.response_name = root.tag
+        self.status = root.get("status")
+        self.status_text = root.get("status_text")
+
+
+@dataclass
+class AuthenticateResponse(Response):
+    """
+    Response Object for authenticate command
+    """
+
+    role: Role
+    timezone: str
+    severity: str
+
+    def __init__(self, gmp, root: etree.Element):
+        super().__init__(root)
+        self.role = Role.resolve_role(root.find("role"))
+        self.timezone = root.find("timezone").text
+        self.severity = root.find("severity").text
+
+
+@dataclass
+class GetPortListsResponse(Response):
+    """
+    Response Object for a get_port_lists command
+    """
+
+    port_lists: list
```

Comment:
It would be better to use the List from https://docs.python.org/3/library/typing.html The List from typing can get a type of the contained objects.
```suggestion
    port_lists: List[PortList]
```